### PR TITLE
ID shifts for Dec 10th client update.

### PR DESCRIPTION
**Pull request by wrenffxi**

### CODE REVIEW

**Comment by zynjec** [comment]
Missed updating the actual version number!

File: scripts/zones/La_Theine_Plateau/IDs.lua [hunk 1]
```diff
@@ -48,8 +48,8 @@ zones[dsp.zone.LA_THEINE_PLATEAU] =
     },
     npc =
     {
-        FALLEN_EGG = 17195582,
-        RAINBOW    = 17195606,
+        FALLEN_EGG = 17195583,
```
[the remark this answers is not in the document]
Comment (nfaletra) [reply scripts/zones/La_Theine_Plateau/IDs.lua]: ID in npc_list.sql for this is correct.

File: scripts/zones/La_Theine_Plateau/IDs.lua [hunk 2]
```diff
@@ -48,8 +48,8 @@ zones[dsp.zone.LA_THEINE_PLATEAU] =
...
+        RAINBOW    = 17195607,
```
[the remark this answers is not in the document]
Comment (nfaletra) [reply scripts/zones/La_Theine_Plateau/IDs.lua]: ID in npc_list.sql for this is correct.

**Comment by nfaletra** [comment]
I should say those IDs DID change, but they were somehow wrong in the script before? I must have fat fingered it last update or something.

File: scripts/zones/La_Theine_Plateau/IDs.lua (hunk 2 above)
[the remark this answers is not in the document]
Comment (wrenffxi) [reply scripts/zones/La_Theine_Plateau/IDs.lua]: fixed

**Pull request merged**